modules/RTC/TPCUtils.js

```diff
@@ -316,7 +316,7 @@ export class TPCUtils {
     replaceTrack(oldTrack, newTrack) {
         if (oldTrack && newTrack) {
             const mediaType = newTrack.getType();
-            const stream = newTrack.getOriginalStream();
+            const stream = newTrack.stream;
```

Comment:
`.stream` is not really public and should use `getOriginalStream` getter

Reply:
This wasn't an intentional change, was debugging something and forgot to take it out. Thanks for pointing it out.